.github/workflows/scripts/comments.mjs

```diff
@@ -1,14 +1,70 @@
+import { readFile } from 'node:fs/promises'
+
+/**
+ * Posts the content of a diff as a comment on a Github issue
+ *
+ * @param {GithubActionContext} githubActionContext
+ * @param {number} issueNumber
+ * @param {DiffComment} diffComment
+ */
+export async function commentDiff (
+  githubActionContext,
+  issueNumber,
+  { path, title, markerText }
+) {
+  // Read diff from previous step
+  const diffText = await readFile(path, 'utf8')
+
+  // Add or update comment on PR
+  try {
+    await comment(
+      githubActionContext,
+      issueNumber,
+      markerText,
+      renderDiffBodyText(title, diffText)
```

Comment:
I found this a bit hard to follow but [added some suggestions](https://github.com/alphagov/govuk-frontend/pull/3671/files#r1202654624)